sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -445,7 +400,7 @@ public boolean hasInternalKey() {
       }
 
       public static final int NAME_FIELD_NUMBER = 4;
-      private java.lang.Object name_;
+      private volatile java.lang.Object name_;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "name_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)